### ISSUE TITLE
responsive height

### ISSUE THREAD

**Issue by tcurdt** [kenwheeler/slick#1024]
This looks like a great plugin. I'd be interested in a combination of the "responsive" and "variable width" demo use case - but I noticed that on the responsive example the height doesn't change at all.

...and I don't see if that's possible at all.

**Comment by kenwheeler** [comment]
Variablewidth doesn't scale the widths of the items. How would that work? Proportional scale?

**Comment by tcurdt** [comment]
Yup. So basically I would define a height for each breakpoint and then the width of would be scaled proportionally.

For me personally it would be even better if the first image would just always be 100% of the width of the parent container - but I don't think this could be generalized. So I guess giving the height per breakpoint could be a more generally useful approach.

Does that make sense?

**Comment by kenwheeler** [comment]
Yeah it's tough though with the 98% approach. For something to be included in core it has to be widely applicable. Also, that's a lot of calculation, given that each slide has different proportions in a variable width scenario

**Comment by virtual** [comment]
Hi, I've been working on updating my centered slider to work with 1.4.1, if this is any help to you. I've found that I needed to add another breakpoint, otherwise a large gap is forced below the slider for some viewpoints.
1.4.1: http://jsfiddle.net/Virtual/nzrgd8bc/
1.3.15: http://jsfiddle.net/Virtual/jqxW8/

**Comment by tcurdt** [comment]
@kenwheeler But having a different height on mobile and desktop should be a pretty common use case, no?

@virtual Those fiddles are a great start! Thanks a lot!

**Comment by kenwheeler** [comment]
@tcurdt variable width is really he only place it shows up. And Height and width don't really work the same on the web

**Comment by tcurdt** [comment]
> variable width is really he only place it shows up

@kenwheeler because otherwise the responsive just works you mean?

I am basically after http://jsfiddle.net/tcurdt/csfpxyou/8/ that works for reponsive just like this http://jsfiddle.net/tcurdt/fj87bope/

**Comment by desandro** [comment]
You might want to try [Flickity](http://flickity.metafizzy.co). Flickity uses the sizing you set with your own CSS. With [`setGallerySize: false`](http://flickity.metafizzy.co/options.html#setgallerysize), you can size the height of the gallery proportionally based on its width. See demo: http://codepen.io/desandro/pen/dPmKyx

**Comment by simeydotme** [comment]
Yo @desandro ... I highly doubt that trying to ride Slick's popularity and pimping another plugin in the issue tracker is helping to fix the issues ... it's great that you feel the desire to contribute, but why don't you put that passion in to helping @kenwheeler and his users to fix these bugs ? :smile: 


Pretty low.

**Comment by kenwheeler** [comment]
@simeydotme it's all good, he has my permission. There are some things that slick just can't do in current form. There are also some things Flickity can't do in current form that slick does. So it's all good.

**Comment by simeydotme** [comment]
:)
fair enough - just concern that anyone with heavy investment might get the wrong idea .... I know I don't want the hassle of someone seeing these comments and asking me to replace slick across my projects :stuck_out_tongue_closed_eyes: :joy:
